### ISSUE TITLE
New ISOM 2017 Appendix Published – CMYK Printing and Colour Definitions

### ISSUE THREAD

**Issue by Symbian9** [OpenOrienteering/mapper#1106]
### Actual behaviour

IOF published in official blog:

> The process of making/drawing orienteering maps has changed completely in the last 25 years, but the recommended printing method for maps has stayed the same. However, the IOF has now updated the recommendations according to modern practices.
> 
> The IOF Map Commission have thus prepared an Appendix to ISOM 2017, which explains the background of the update, and the different printing techniques. You can read the full document here: ISOM 2017 Appendix 1 – CMYK Printing and Colour Definitions](https://orienteering.org/wp-content/uploads/2018/06/ISOM-2017-Appendix-1-CMYK-Printing-and-Colour-Definitions.pdf).

### Expected behaviour

Update CMYK values of colors in ISOM2017 symbol sets distributed with OpenOrienteering Mapper.

- https://orienteering.org/wp-content/uploads/2018/06/ISOM-2017-Appendix-1-CMYK-Printing-and-Colour-Definitions.pdf

### Configuration

**Mapper Version**:
0.9.x

**Operating System**:
all

### References

- http://orienteering.org/new-isom-2017-appendix-published-cmyk-printing-and-colour-definitions/

**Comment by Symbian9** [comment]
*IOF Council Meeting Minutes 189* Published

- http://orienteering.org/iof-council-meeting-minutes-published-3/
  - PDF - https://orienteering.org/wp-content/uploads/2018/06/Minutes-of-Council-Minutes-189.pdf

> ...
> 20 Report from MC
> 
> ...
> 20.2 Print Appendix for ISOM 2017
> 
> TH reported on behalf of the MC that the Print Appendix to ISOM 2017 had received very few comments during the consultation period, and after some minor corrections was now ready for approval. Council approved the Print Appendix for publication.
> ...

**Comment by Symbian9** [comment]
Presentations from 18th ICOM now available for downloading as PDFs:

- https://orienteering.org/presentations-from-icom-2018/

Here is continuous discussion on *«ISOM 2017 Appendix 1 – CMYK Printing and Colour Definitions»* (PDF)

- https://orienteering.org/wp-content/uploads/2018/10/ICOM2018_Carlsson_Discussion_on_ISOM2017_Appendix1_CMYK_printing_and_colour_definitions.pdf

**Comment by dg0yt** [comment]
> Here is continuous discussion on «ISOM 2017 Appendix 1 – CMYK Printing and Colour Definitions» (PDF)

That is not so much a discussion but the position of the appendix' author. Even while there was almost no time for discussion after this talk at ICOM 2018, it became obvious that some points in this appendix are quite controversial.

**Comment by Symbian9** [comment]
Yep, here is basic visual comparsion of "ISOM 2017" and "ISOM 2017, Appendix 1"

- https://www.facebook.com/groups/485564718218028?view=permalink&id=1892641680843651

**Comment by ollesmaps** [comment]
@dg0yt You were at ICOM :open_mouth:   I did not connect your face to your online presense. Otherwise I would personally say thank you for your OOMapper efforts! So this online thank you! must do for now :smile: 
(Sorry for offtopic)

**Comment by Symbian9** [comment]
IOF MC finally updated and released *"ISOM 2017 Appendix 1" (Version 1, 2019-06-17)*, including colours for `ISOM 2017-2` and `ISSprOM 2019`.

Also, released test sheet in OCD11 format (`PrintTestSheet_ver20190412_ocd11.ocd`) and instruction for it as PDF (`PrintTech2019_Instructions.pdf`).

But when I try open `PrintTestSheet_ver20190412_ocd11.ocd` in Mapper `unstable-0.9.20190604` it give me next warning message:

> ### Warning
>
> **The map import generated warnings.**
>
> In line symbol 306.1 'Minor/Seasonal Water Channel': The dash pattern's end length (0.75 mm) cannot be imported correctly. Using 0.625 mm.
>
> In line symbol 103.0 'Help contour': The dash pattern's end length (0.7 mm) cannot be imported correctly. Using 0.625 mm.
> 
> In line symbol 509.4 'Narrow Ride': The dash pattern's end length (2 mm) cannot be imported correctly. Using 1.5 mm.
>
> In line symbol 530.4 'Ruin': The dash pattern's end length (0.62 mm) cannot be imported correctly. Using 0.75 mm.

**Comment by ollesmaps** [comment]
There is a new Appendix. And as I read it there are some slight changes in order of colors.

**Comment by Symbian9** [comment]
Just compared update to *ISSprOM 2019* colors

| `Colours ISSprOM 2019.pdf` (previous; draft) | `ISOM 2017 Appendix 1 CMYK Printing and Colour Definitions (temporary including ISprOM) Version 3` (actual) |
| --- | --- |
| Upper Purple for Course Overprint | Purple 100% (all course planning symbols exept 701, 703, 705, 706, 708, 710,1) |
| White for Course Planning Symbols | White (Background to control descriptions and white shade for control numbers) |
| Black 100% | Black 100% |
| Green 100% - Point feature | Green 100% (point symbols) |
| Lower Purple for Course Planning Symbols | Purple 100% (course planning symbols mentioned above) | 
| Purple 50% | Purple 50% |
| Brown 100% - North Line | |
| Blue 100% - Line feature | Blue 100% |
| Brown 100% - Line feature | Brown 100% |
| Black 60% - Area feature | Black 60% (buildings) |
| | Black 50% (tramway) |
| Black 20% - Area feature | Black 20% (canopy) |
| Blue 100% - Area feature | |
| Blue 70% - Area feature | Blue 70% |
| Blue 30% - Area feature | Blue 30% |
| White for Green/Brown | |
| Brown 50% - Area feature(will ev. be changed) | Brown 50% |
| Brown 30% (upper)(will ev. be changed) | Brown 30% |
| | Green 50%+yellow 100% |
| Green 100% - Area feature | Green 100% (reference shade 86_0_91_0) |
| Yellow 100%/Green 50% | |
| Green 60% - Area feature | Green 60% |
| Green 30% - Are feature | Green 30% |
| Green 100% - Undergrowth | | 
| White for Yellow | |
| Black Cultivated Land | Black 30% (bare rock) |
| | Yellow 100% |
| | Yellow 50% |
| Brown (Area)(will ev. be changed) | |
